website/public/js/services/notificationServices.js

```diff
@@ -77,8 +77,24 @@ angular.module("habitrpg").factory("Notification",
     streak: function(val) {
       notify(window.env.t('streakName') + ': ' + val, 'streak', 'glyphicon glyphicon-repeat');
     },
-    drop: function(val) {
-      notify(val, 'drop', 'glyphicon glyphicon-gift');
+    drop: function(val, item) {
+      if ( item !== undefined ) {
+       var dropClass = "";
```

Comment:
This should be one line higher, so it's out of the if block, since it's called on line 97 by the notify function.